.gitignore

```diff
@@ -0,0 +1,11 @@
+# JetBrains
+.idea
+
+# PyCache
+__pycache__
+
+# Python Development Environment
+build
+dist
+src/cnbc.egg-info
```

Comment:
should this be src/yh_finance.egg-info ?

Reply:
Yes. I'll apply this in the next round of revisions.